model/atmosphere/advection/src/icon4py/model/atmosphere/advection/stencils/compute_ffsl_backtrajectory_counterclockwise_indicator.py

```diff
@@ -0,0 +1,46 @@
+# ICON4Py - ICON inspired code in Python and GT4Py
+#
+# Copyright (c) 2022-2024, ETH Zurich and MeteoSwiss
+# All rights reserved.
+#
+# Please, refer to the LICENSE file in the root directory.
+# SPDX-License-Identifier: BSD-3-Clause
+
+import gt4py.next as gtx
+from gt4py.next.ffront.fbuiltins import broadcast, where
+
+from icon4py.model.common import dimension as dims, field_type_aliases as fa, type_alias as ta
+from icon4py.model.common.settings import backend
+
+
+@gtx.field_operator
+def _compute_ffsl_backtrajectory_counterclockwise_indicator(
+    lcounterclock: bool,
+    p_vn: fa.EdgeKField[ta.wpfloat],
+    tangent_orientation: fa.EdgeField[ta.wpfloat],
+) -> fa.EdgeKField[bool]:
+    tangent_orientation = broadcast(tangent_orientation, (dims.EdgeDim, dims.KDim))
```

Comment:
`broadcast` is not needed as long as horizontal dimensions match, i.e. gt4py does it in the background for KDim already